LondonBritishLibrary/orient/BLorient765.xml

```diff
@@ -30,20 +30,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
```

Comment:
I guess you have committed this manuscript to another branch already... Am I right?

Reply:
If I understand correctly you have merged master, so that your changes are undone and you will work on it in another branch. Am I right?